guides/kubernetes.md

```diff
@@ -0,0 +1,148 @@
+# Kubernetes guidance
```

Comment:
Could we have a quick introduction just to clarify who/what this guidance is for?